djangocms_video/forms.py

```diff
@@ -2,14 +2,15 @@
 import re
 
 from django import forms
+from django.conf import settings
 
 from . import models
 
 
 YOUTUBE_URL_RE = re.compile(r'(?:(?:http://|https://|//)?(?:www\.)?youtu\.?be.*).*')
 # https://stackoverflow.com/a/9102270
 YOUTUBE_VIDEO_ID_RE = re.compile(r'(?:[?&]v=|/embed/|/1/|/v/|https?://(?:www\.)?youtu\.be/)([^&\n?#]+)')
-YOUTUBE_EMBED_URL = '//www.youtube.com/embed/{}'
+DEFAULT_YOUTUBE_EMBED_URL = '//www.youtube.com/embed/{}'
```

Comment:
please document this in the readme :)

Reply:
Also an entry to the changelog

Reply:
@FinalAngel I added a README and a CHANGELOG section